nginx/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM nginx
+
+MAINTAINER Tamer Mohamed Bahgat
+
+COPY nginx.conf:/etc/nginx/nginx.conf
```

Comment:
@TamerB I'm getting this error while building with docker-compose. The `COPY` command is failing. Please fix and push.

![image](https://user-images.githubusercontent.com/611426/28787777-d69fd7e4-761d-11e7-91f1-85133972abd9.png)